app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -387,8 +359,9 @@ private void initSpinner() {
             public void onItemSelected(final AdapterView<?> parent, final View view, final int position, final long id) {
 
                 Chapter selectedChapter = mChapterAdapter.getItem(position);
-                updateSelectionfor(selectedChapter);
+                updateSelectionfor(selectedChapter.getGplusId());
```

Comment:
I wonder if we should keep a reference to the previous selected chapter and in case we the user for some reason reselects the same chapter we avoid doing a page reload again.